prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/FL/FL-receiver-transform.yml

```diff
@@ -0,0 +1,65 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: ks-name-of-the-sender-application-Namespace-Id
+    condition: true
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ 'MSH-3-1' ]
+
+  - name: ks-sender-application-Universal-Id
+    condition: true
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ 'MSH-3-2' ]
+
+  - name: ks-sender-application-Universal-Id-Type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-3-3' ]
+
+  - name: ks-name-of-the-receiving-application
+    condition: true
+    value: [ '"FDOH-ELR"' ]
+    hl7Spec: [ 'MSH-5-1' ]
+
+  - name: ks-OID-of-the-receiving-application
+    condition: true
+    value: [ '"2.16.840.1.114222.4.3.3.8.1.3"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: ks-indicates-that-5.2-is-an-OID
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
+  - name: ks-receiving-facility-namespace-ID
+    condition: true
+    value: [ '"FDOH"' ]
+    hl7Spec: [ 'MSH-6-1' ]
+
+  - name: ks-receiving-facility-universal-ID
+    condition: true
+    value: [ '"2.16.840.1.114222.1.3645"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: ks-receiving-facility-universal-ID-Type
+    condition: true
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
+  - name: ks-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: order-note
```

Comment:
What is this doing?

Reply:
It add AOE to OBX instead of AOE segment.